apps/wallet-dashboard/components/Dialogs/Staking/StakeDialog.tsx

```diff
@@ -220,6 +219,18 @@ export function StakeDialog({
                             showActiveStatus
                         />
                     )}
+                    {view === StakeDialogView.TransactionDetails && (
```

Comment:
just a comment: Why had you put it in the previous condition and not here? 
`... && validatorApy && (`

Reply:
right, cause there was no need for it to be always present, or it could be null